src/components/email-confirm-modal/email-confirm-modal.js

```diff
@@ -0,0 +1,93 @@
+import { Button } from '@mui/material'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { style } from './email-confirmation-modal.style'
+import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import imgSuccess from '~/assets/img/email-confirmation-modals/success-icon.svg'
+import imgReject from '~/assets/img/email-confirmation-modals/not-success-icon.svg'
+import LoginDialog from '~/containers/guest-home-page/login-dialog/LoginDialog'
+import useAxios from '~/hooks/use-axios'
+import { AuthService } from '~/services/auth-service'
+import Loader from '~/components/loader/Loader'
+
+const EmailConfirmModal = ( { confirmToken, setModal } ) => {
+
+  const { t } = useTranslation()
+
+  const serviceFunction = useCallback(() => AuthService.confirmEmail(confirmToken), [confirmToken])
+
+  const { response, error, loading } = useAxios( { service: serviceFunction } )
+
+  const openLoginDialog = () => {
+    setModal(<LoginDialog />)
+  }
+
+  if (loading) {
+    return (
+      <Box>
+        <Loader size={ 100 } />
+      </Box>
+    )
+  }
+
+  if (error) {
+    return (
+      <Box sx={ style.box }>
+        <Box>
+          <Box
+            alt="email-reject-icon"
+            component="img"
+            src={ imgReject }
+          />
+          <Typography
+            component="h2" data-testid='reject-message' id="modal-modal-title"
```

Comment:
For what you use id="modal-modal-title" ?

In line 73 you also have this id, but it never used in your code.

And i also think about naming

Reply:
fixed